pypackage/{{ cookiecutter.slug }}/src/{{ cookiecutter.package_name }}/cli.py

```diff
@@ -2,12 +2,13 @@
 from importlib.metadata import version
 
 
-def cli(_argv=None):  # pylint:disable=inconsistent-return-statements
+def cli(argv=None):
     parser = ArgumentParser()
-    parser.add_argument("-v", "--version", action="store_true")
-
-    args = parser.parse_args(_argv)
-
-    if args.version:
-        print(version("{{ cookiecutter.slug }}"))
-        return 0
+    parser.add_argument(
+        "-v",
+        "--version",
+        action="version",
+        version=version("{{ cookiecutter.slug }}"),
+    )
+
+    _args = parser.parse_args(argv)
```

Comment:
Having to name `_args` with a leading underscore here because otherwise pylint complains about an unused variable. I do want the `cli.py` generated by the cookiecutter to call `parse_args()` because I want it to tee everything up nicely for the user to come along and add the code that is actually going to use `_args`. But the cookiecutter itself doesn't have anything to do with `_args`. The alternative would have been a pylint disable comment.